addons/skin.estuary/xml/DialogPVRChannelGuide.xml

```diff
@@ -75,18 +75,27 @@
 							<visible>!String.IsEmpty(ListItem.Plot) + Control.HasFocus(11)</visible>
 							<animation effect="fade" start="0" end="100" time="200">Focus</animation>
 							<control type="image">
-								<left>782</left>
+								<left>772</left>
 								<top>-100</top>
-								<width>900</width>
+								<width>1210</width>
 								<height>350</height>
 								<texture border="40">buttons/dialogbutton-nofo.png</texture>
 							</control>
+							<control type="image">
+								<left>805</left>
+								<top>-65</top>
+								<width>320</width>
+								<height>280</height>
+								<aspectratio align="center" aligny="center">keep</aspectratio>
+								<texture fallback="$INFO[Listitem.Icon]">$INFO[Listitem.EpgEventIcon]</texture>
```

Comment:
This make no sense as `Listitem.Icon` called on an epg event item has an automatic fallback to the channel icon.
A simple `<texture>$INFO[ListItem.Icon]</texture>` has the same effect.

Reply:
ok, I will change so that if fallback is active don't change it.